### PR TITLE
[FR] Localize `Mutual Transport Layer Security (mTLS)`

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
Localize `Mutual Transport Layer Security (mTLS)` in French

### Related issue number or link (ex: `resolves #issue-number`)
solves https://github.com/cncf/glossary/issues/2647

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/mutual-transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Mutual Transport Layer Security (mTLS)
```
[the remark this answers is not in the document]
Comment (Krast76) [reply content/fr/mutual-transport-layer-security.md:2]: Je suis pour, avec mTLS entre parenthèse ?

[the remark this answers is not in the document]
Comment (sestegra) [reply content/fr/mutual-transport-layer-security.md:2]: Pour les mêmes raison que TLS, je propose "Sécurité mutuelle de la couche de transport (mTLS)"

**Review by sestegra — CHANGES_REQUESTED** [review]
Super, j'ai découvert le terme `on-path attack` qui remplace le terme non inclusif de main-in-the-middle.

**Review by sestegra — APPROVED** [review]
🚀 LGTM

**Review by Krast76 — APPROVED** [review]
LGTM 👍

**Pull request merged**